src/System.Drawing.Common/src/System/Drawing/Design/IPropertyValueUIService.cs

```diff
@@ -1,64 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@safern, @ericstj, lots of these System\Drawing\Design\* files are in the repo but aren't being compiled in.  Ok to delete them or will they be needed?

Reply:
I brought back all of these as they're being consumed into Mono.